src/reports/report.php

```diff
@@ -95,6 +97,7 @@ public function post()
 			}
 		}
 
+		$this->plugin->customAttributes->set("total", $reportPlugin->getTotal());
```

Comment:
Don't think this is a good idea. We can't expect developers to know this internal attribute named `customAttributes`

Needs to be `$this->plugin->setCustomAttribute($key, $value)`